src/main/java/com/github/dockerjava/api/model/Container.java

```diff
@@ -75,6 +84,12 @@
     private ContainerHostConfig hostConfig;
 
     /**
+     * @since ~{@link RemoteApiVersion#VERSION_1_23}
+     */
+    @JsonProperty("Mounts")
+    private List<InspectContainerResponse.Mount> mounts;
```

Comment:
inspect response has different from list container mount information. 
Probably new object should be create for this response...

Reply:
So what do you think? I can change it to new object if you want, but I don't know what it should be as they look the same to me.

Reply:
We mistakenly used (HostConfig AFAIR) in two places and because of different options from two places object become hardly understandable. It not the end of the world of course, but I usually checking also docker sources for structs/types.